_topic_maps/_topic_map.yml

```diff
@@ -3392,6 +3392,11 @@ Topics:
   File: psap-node-feature-discovery-operator
 - Name: Kernel Module Management Operator
   File: kmm-kernel-module-management
+- Name: Kernel Module Management Operator release notes
+  Dir: release-notes
+  Topics:
+  - Name: Release notes for Kernel Module Management Operator 2.2
```

Comment:
I'm just thinking out loud here: wouldn't there be an easier way to do this? It looks like if we continue with this route, every release would require a new entry into the topic map. Why not created a "main" release notes file that serves as the assembly, and then create modules for each release? Or create a "main" release notes file for the assembly, and then include 1 module that uses headings/subheadings for each release like we do in OCP? 

I'm fine with merging this for now, and you probably have your reasons. But maybe reconsider how future entries are going to look with this format?